### ISSUE TITLE
custom shaders (images, shadows)

### ISSUE THREAD

**Issue by cztomsik** [servo/pathfinder#221]
Is it possible to render regular bitmap images? Or is it possible to use custom shader somehow?

Also, I've noticed, there's `canvas.set_shadow_offset()` but from what I've understood, it does not do any blurring, it's just an outline, am I right? If so, is this planned?

**Comment by pcwalton** [comment]
Yes, both of these are planned. We can't ship a canvas implementation without them :)

**Comment by pcwalton** [comment]
We now have images and shadow blurring. I'll leave this open for the possibility of custom shaders.

**Comment by ChameleonScales** [comment]
I'm curious, is there a size limit for bitmap images ?
For example cairo has a hard limit of 2^15 = 32 768 px.
Can pathfinder handle bigger images?
